theme_ipsum/static/src/js/editor.js

```diff
@@ -0,0 +1,36 @@
+odoo.define("theme_ipsum.editor", function(require) {
+    "use strict";
+    var options = require("web_editor.snippets.options");
+    var LinkDialog = require("wysiwyg.widgets.LinkDialog");
+
+    /* This exits for the only purpose to be able to duplicate the custom snippets */
+    options.registry.snippet_ipsum = options.Class.extend({
+        onFocus: function() {
+            this.$el
+                .parent()
+                .find(".oe_snippet_clone")
+                .removeClass("d-none");
```

Comment:
This was a work-around and removed it because it's not necessary anymore. I've added `data-drop-near` attributes so the snippets are now marked as movable which makes the clone button appear.